sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateService.java

```diff
@@ -0,0 +1,401 @@
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.certificates;
+
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceModifiedException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.VoidResponse;
+import com.azure.core.implementation.annotation.BodyParam;
+import com.azure.core.implementation.annotation.Delete;
+import com.azure.core.implementation.annotation.ExpectedResponses;
+import com.azure.core.implementation.annotation.Get;
+import com.azure.core.implementation.annotation.HeaderParam;
+import com.azure.core.implementation.annotation.Host;
+import com.azure.core.implementation.annotation.HostParam;
+import com.azure.core.implementation.annotation.Patch;
+import com.azure.core.implementation.annotation.Post;
+import com.azure.core.implementation.annotation.Put;
+import com.azure.core.implementation.annotation.PathParam;
+import com.azure.core.implementation.annotation.QueryParam;
+import com.azure.core.implementation.annotation.ReturnValueWireType;
+import com.azure.core.implementation.annotation.ServiceInterface;
+import com.azure.core.implementation.annotation.UnexpectedResponseExceptionType;
+import com.azure.core.util.Context;
+import com.azure.security.keyvault.certificates.implementation.CertificateBasePage;
+import com.azure.security.keyvault.certificates.implementation.ContactPage;
+import com.azure.security.keyvault.certificates.implementation.DeletedCertificatePage;
+import com.azure.security.keyvault.certificates.implementation.IssuerBasePage;
+import com.azure.security.keyvault.certificates.models.Certificate;
+import com.azure.security.keyvault.certificates.models.CertificateBase;
+import com.azure.security.keyvault.certificates.models.CertificateOperation;
+import com.azure.security.keyvault.certificates.models.DeletedCertificate;
+import com.azure.security.keyvault.certificates.models.Contact;
+import com.azure.security.keyvault.certificates.models.Issuer;
+import com.azure.security.keyvault.certificates.models.IssuerBase;
+import com.azure.security.keyvault.certificates.models.CertificatePolicy;
+import reactor.core.publisher.Mono;
+
+/**
+ * The interface defining all the services for {@link CertificateAsyncClient} to be used
+ * by the proxy service to perform REST calls.
+ *
+ * This is package-private so that these REST calls are transparent to the user.
+ */
+@Host("{url}")
+@ServiceInterface(name = "KeyVault")
+interface CertificateService {
+
+    @Post("certificates/{certificate-name}/create")
+    @ExpectedResponses({202})
+    @UnexpectedResponseExceptionType(code = {400}, value = ResourceModifiedException.class)
+    @UnexpectedResponseExceptionType(HttpResponseException.class)
+    Mono<Response<CertificateOperation>> createCertificate(@HostParam("url") String url,
```

Comment:
Why doesn't this API include `Context` param?

Reply:
It was part of polling API, so didn't add it as sync client can't directly call this method.
But logically it can be added for internal async side, so added the context parameter.

But, in general Polling API tracing is not yet supported.